riotdocker-base/build.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Automatically exit on error
+set -e
+
+COUNTER_STEP=0
+COUNTER_SUBSTEP=0
+BLUE="\e[34m"
+BOLD="\e[1m"
+NORMAL="\e[0m"
+
+step() {
+    COUNTER_SUBSTEP=0
+    COUNTER_STEP=$(("$COUNTER_STEP" + 1))
+    if [ -n "${GITHUB_RUN_ID}" ]; then
+        echo "::group::"
+    fi
+    printf "${BLUE}${BOLD}==>${NORMAL}${BOLD} Step %d:${NORMAL} %s\n" "$COUNTER_STEP" "$1"
+}
+
+endstep() {
+    if [ -n "${GITHUB_RUN_ID}" ]; then
```

Comment:
This apparently doesn't work. It actually is reassuring that environment variables do not leak into the container; but now I have no idea how to emit the `::group::` ... `::endgroup::` only when run in Github.

Reply:
Yeah, if only they did the regular thing, advertised that they are their own TERM, or just packed these in vendor specific OSC codes.

Might be that the Docker environment would need to pass on the variable – but unless you particularly feel like it, let's not spend more time on this.